widgets/src/graph/Graph.js

```diff
@@ -111,19 +110,25 @@
         return this;
     };
 
-    Graph.prototype.setZoom = function (translation, scale) {
+    Graph.prototype.setZoom = function (translation, scale, transitionDuration) {
         if (this.zoom) {
             this.zoom.translate(translation);
             this.zoom.scale(scale);
-            this.applyZoom();
+            this.applyZoom(transitionDuration);
         }
     };
 
-    Graph.prototype.applyZoom = function (transition) {
-        (transition ? this.svg.transition() : this.svg)
+    Graph.prototype.applyZoom = function (transitionDuration) {
+        if (d3.event && d3.event.sourceEvent && (!d3.event.sourceEvent.ctrlKey && d3.event.sourceEvent.type == 'mousewheel' || d3.event.sourceEvent.type == 'DOMMouseScroll')) {
```

Comment:
Are we currently supporting tablet-based scrolling with the above two sourceevent type checks? Would it be better to check the WheelEvent interface?